src/color/creating_reading.js

```diff
@@ -111,171 +118,170 @@ p5.prototype.brightness = function(c) {
- * strings and all named color strings are supported. In this case, an alpha
- * number value as a second argument is not supported, the RGBA form should be
- * used.
+ * Creates a <a href="#/p5/p5.Color">p5.Color</a> object. By default, the
+ * parameters are interpreted as RGB values. Calling `color(255, 204, 0)` will
+ * return a bright yellow color. The way these parameters are interpreted may
+ * be changed with the <a href="#/p5/colorMode">colorMode()</a> function.
+ *
+ * The version of `color()` with one parameter interprets the value one of two
+ * ways. If the parameter is a number, it's interpreted as a grayscale value.
+ * If the parameter is a string, it's interpreted as a CSS color string.
+ *
+ * The version of `color()` with two parameters interprets the first one as a
+ * grayscale value. The second parameter sets the alpha (transparency) value.
+ *
+ * The version of `color()` with three parameters interprets them as RGB, HSB,
+ * or HSL colors, depending on the current `colorMode()`.
+ *
+ * The version of `color()` with four parameters interprets them as RGBA, HSBA,
+ * or HSLA colors, depending on the current `colorMode()`. The last parameter
+ * sets the alpha (transparency) value.
  *
  * @method color
  * @param  {Number} gray number specifying value between white and black.
  * @param  {Number} [alpha] alpha value relative to current color range
- *                                 (default is 0-255)
- * @return {p5.Color} resulting color
+ *                                 (default is 0-255).
+ * @return {p5.Color} resulting color.
  *
  * @example
  * <div>
  * <code>
- * let c = color(255, 204, 0);
+ * const c = color(255, 204, 0);
  * fill(c);
  * noStroke();
  * rect(30, 20, 55, 55);
- * describe(`Yellow rect in middle right of canvas,
- * with 55 pixel width and height.`);
+ * describe('A yellow rectangle on a gray canvas.');
  * </code>
  * </div>
  *
  * <div>
  * <code>
+ * // RGB values.
  * let c = color(255, 204, 0);
  * fill(c);
  * noStroke();
- * ellipse(25, 25, 80, 80); // Draw left circle
- * // Using only one value generates a grayscale value.
+ * circle(25, 25, 80);
+ * // A grayscale value.
  * c = color(65);
  * fill(c);
- * ellipse(75, 75, 80, 80);
- * describe(`Yellow ellipse in top left of canvas, black ellipse in bottom
- * right, both 80×80.`);
+ * circle(75, 75, 80);
+ * describe(
+ *   'Two ellipses. The circle in the top-left corner is yellow and the one at the bottom-right is gray.'
+ * );
  * </code>
  * </div>
  *
  * <div>
  * <code>
- * // You can use named SVG & CSS colors
- * let c = color('magenta');
+ * // A CSS named color.
+ * const c = color('magenta');
  * fill(c);
  * noStroke();
- * rect(20, 20, 60, 60);
- * describe('Bright fuchsia rect in middle of canvas, 60 pixel width and height.');
+ * square(20, 20, 60);
+ * describe('A magenta square on a gray canvas.');
  * </code>
  * </div>
  *
  * <div>
  * <code>
- * // Example of hex color codes
+ * // CSS hexadecimal color codes.
```

Comment:
Might be worth adding (hex) in brackets in case people are googling these and don't know they refer to the same thing?